### DISCUSSION TITLE
Are some exceptions not handled on purpose?

### DISCUSSION THREAD

**Discussion by elenaoat** [pinax/pinax-stripe#656] [Q&A]
validate method of Event model class retrieves an event from Stripe using API. No exception cases are handled in this case (and in other ones too, I noticed). Is this done on purpose? There are cases, in which an exception would be raised. For instance, if an application is deauthorized, an event is sent. However, this event cannot be retrieved and an exception is raised. Any comments on this?

**Comment by paltman** [comment]
@elenaoat thanks for raising this issue (sorry, couldn't resist the use of of a slight pun there :smile:).  I think the larger question is what do we do when handling the various exceptions. when in doubt, i'd like to just see the exception bubble up and only then start writing code to catch them if there is something i can do in response. do you have anything specific in mind?
